website/docs/d/org.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org"
+sidebar_current: "docs-vcd-data-source-org"
+description: |-
+  Provides an organization data source.
+---
+
+# vcd\_org
+
+Provides a vCloud Director Org data source. An organization can be used to manage catalogs, virtual
+data centers, and users.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_org" "my-org" {
+  name   = "my-org"
+}
+
+resource "vcd_org" "my-org-clone" {
+  name                 = "my-org-clone"
+  full_name            = "${data.vcd_org.my-org.full_name}"
+  can_publish_catalogs = "${data.vcd_org.my-org.can_publish_catalogs}"
+  deployed_vm_quota    = "${data.vcd_org.my-org.deployed_vm_quota}"
+  stored_vm_quota      = "${data.vcd_org.my-org.stored_vm_quota}"
+  is_enabled           = "${data.vcd_org.my-org.is_enabled}"
+  delete_force         = "true"
+  delete_recursive     = "true"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Org name
+* `full_name` - (Optional) Org full name
```

Comment:
This is general question for LB documentation too. We saying attributes `optional`, but in reality if user would provide we don't care cause we fetch only by name, but still may don't match what we get from vCD ? ^^ @Didainius

Reply:
I left only the fields which are allowed to be input in `Argument` section. Everything else should probably in `Attribute reference` section. I have left it so for load balancer components.
<img width="783" alt="image" src="https://user-images.githubusercontent.com/15804230/62289076-75f27a80-b466-11e9-8465-65ad6d084062.png">

Here's a better example how others do it - https://www.terraform.io/docs/providers/vsphere/d/compute_cluster.html